cpp/include/cudf/io/writers.hpp

```diff
@@ -193,8 +193,8 @@ class writer {
    *
    * @param[in] pq_chunked_state State information that crosses _begin() / write_chunked() / _end() boundaries.   
    */
-  void write_chunked_begin(struct pq_chunked_state& state);                           
-  
+  auto write_chunked_begin() -> std::size_t;
```

Comment:
Why did you switch to the trailing return type in this PR?

Reply:
Same concern

Reply:
Yea, bad habit of mine - I will switch it back. It is just my personal preference and I forget when I am refactoring or coding sometimes.